src/main/java/org/opentripplanner/index/model/PatternShort.java

```diff
@@ -6,21 +6,48 @@
 import org.opentripplanner.routing.edgetype.TripPattern;
 
 import com.beust.jcommander.internal.Lists;
+import org.opentripplanner.util.model.EncodedPolylineBean;
 
 public class PatternShort {
 
     public String id;
     public String desc;
-    
+    public EncodedPolylineBean geometry;
+
+    /**
+     * Constructor for maintaining backwards compatibility
+     */
     public PatternShort (TripPattern pattern) {
         id = pattern.code;
         desc = pattern.name;
+        geometry = null;
```

Comment:
Reduce the amount of logic in constructors by always calling `this` to the constructor with the most arguments:

```suggestion
        this(pattern, null)
```

Reply:
This is really nice thanks! Refactored in 1360e72e44678135324c29f7ac4942514fa05270